test/e2e/framework/framework.go

```diff
@@ -31,7 +31,9 @@ import (
 	"sync"
 	"time"
 
-	"k8s.io/api/core/v1"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
```

Comment:
github.com imports should be in a separate group

Reply:
Ok I will update the PR sometime early this week. Thank you @andrewsykim